phpunit.xml.dist

```diff
@@ -22,14 +22,16 @@
             <directory>./</directory>
             <exclude>
                 <directory>./Tests/</directory>
+                <directory>./DataFixtures/</directory>
                 <directory>./Resources/</directory>
-                <directory>./DependencyInjection/</directory>
                 <directory>./vendor/</directory>
-
-                <!-- Ignore ODM driver -->
```

Comment:
hmm strange comment, can you find out why this was explicitely excluded @jordisala1991?

Reply:
Not sure why it was removed, seems its been here since the begining of this repository.

For me its like excluding entities from the coverage

Reply:
👍